core/src/main/java/com/amplitude/core/utilities/JSON.kt

```diff
@@ -45,6 +45,17 @@ internal fun List<*>?.toJSONArray(): JSONArray? {
     return jsonArray
 }
 
+internal fun Array<*>?.toJSONArray(): JSONArray? {
```

Comment:
Nice. This makes me think we should change the `List<*>?.toJSONArray()` to `Collection<*>?.toJSONArray()` to cover more cases?

Reply:
That's a good catch! Let me just adjust List to Collection then